popgen.md

```diff
@@ -0,0 +1,339 @@
+        u
+        for u in ts.samples()
+        if (
+            ts.population(ts.node(u).population).metadata.get("region") in keep_regions
+            or ts.population(ts.node(u).population).metadata.get("name") == "Denisovan"
+        )
+    ])
+    tszip.compress(ts, "data/unified_genealogy_2q_108Mb-110Mb.tsz")
+
+def create_notebook_data():
+    download_unified_genealogy()
+
+# create_notebook_data()  # uncomment to recreate the tree seqs used in this notebook
+```
+
+(sec_intro_popgen)=
+
+# `Tskit` for population genetics
+
+{ref}`Tskit<tskit:sec_introduction>`, the tree sequence toolkit, brings the power of
+evolutionary trees to the field of population genetics. The
+{ref}`succinct tree sequence<sec_what_is>` format
+is designed to store DNA sequences jointly with their ancestral history (the
+"genetic genealogy" or {ref}`ARG<sec_args>`). Storing population genetic data in this
+form enables highly efficient computation and analysis.
+
+The core `tskit` library provides methods for storing genetic data, a flexible
+analysis framework, and APIs to build your own efficient population genetic algorithms.
+Because of its speed and scalability, `tskit` is well-suited to interactive analysis of
+large genomic datasets. 
+
+## Population genetic simulation
+
+Several simulation tools output tree sequences. Below we use the
+standard library for population genetic simulation models
+([stdpopsim](https://popsim-consortium.github.io/)) to generate a model of
+*Homo sapiens*, in which African, Eurasian,
+and Asian populations combine to generate a mixed American population. We can use the
+[demesdraw](https://pypi.org/project/demesdraw/) package to plot a schematic of the
+migrations and population size changes that define this model. 
+
+
+```{code-cell}
+import stdpopsim
+import demesdraw
+from matplotlib import pyplot as plt
+
+species = stdpopsim.get_species("HomSap")
+model = species.get_demographic_model("AmericanAdmixture_4B11")
+
+# Plot a schematic of the model
+demesdraw.tubes(model.model.to_demes(), ax=plt.gca(), seed=1, log_time=True)
+plt.show()
+```
+
+Genomic data in tree sequence format can be generated via the widely-used
+[msprime](https://tskit.dev/software/msprime.html) simulator. Here we simulate 20kb
+of genome sequence at the start of human chromosome 1 under this model, together with
+its evolutionary history. We generate 16 diploid genomes: 4 from each of the
+populations in the model. The DNA sequences and their ancestry are stored in a succinct
+tree sequence named `ts`:
+
+```{code-cell}
+contig = species.get_contig("chr1", mutation_rate=model.mutation_rate, right=20_000)
+samples = {"AFR": 4, "EUR": 4, "ASIA": 4, "ADMIX": 4} # 16 diploid samples
+engine = stdpopsim.get_engine("msprime")
+ts = engine.simulate(model, contig, samples, seed=9)
+print(f"Simulated a tree sequence of {ts.num_samples} haploid genomes:")
+print(f"{ts.num_sites} variable sites over {ts.sequence_length/1e3} kb of sequence")
```

Comment:
Is `kb` the standard for kilobases? My computer brain reads it as kilobytes.

Reply:
It is pretty standard, but easy enough to spell it out in full, so I've done so in the latest push